internal/root/root_others.go

```diff
@@ -4,9 +4,13 @@ package root
 
 import (
 	"os"
+	"github.com/pkg/errors"
 )
 
 // IsWithSudo tells if a command is runnig with root privileges on
-func IsWithSudo() bool {
-	return os.Getenv("SUDO_UID") != ""
+func IsWithSudo() (bool,error) {
```

Comment:
Since now this function returns error when not running on sudo and nil error when running is sudo, the boolean return value is innecessary and can be removed.

Reply:
Yes,   Sure.